src/evo/creditpool.cpp

```diff
@@ -243,7 +243,7 @@ CCreditPoolDiff::CCreditPoolDiff(CCreditPool starter, const CBlockIndex* pindexP
     assert(pindexPrev);
 
     if (DeploymentActiveAfter(pindexPrev, consensusParams, Consensus::DEPLOYMENT_MN_RR)) {
-        // We consider V20 active if mn_rr is active
+        // No credit pool exists before V20.
```

Comment:
why changing this? we check for mn_rr activation above, not v20..

Reply:
it is a commentary for `fV20Active=true` (next line)
```
platformReward = PlatformShare(GetMasternodePayment(pindexPrev->nHeight + 1, blockSubsidy, /*fV20Active=*/ true));
```